src/lib/y2storage/blk_device.rb

```diff
@@ -33,27 +33,75 @@ class BlkDevice < Device
     storage_class_forward :all, as: "BlkDevice"
     storage_class_forward :find_by_name, as: "BlkDevice"
 
+    # @!method name
+    #   @return [String] kernel-style device name
+    #     (e.g. "/dev/sda2" or "/dev/vg_name/lv_name")
     storage_forward :name
+
+    # @!method name=
+    #   @see #name
     storage_forward :name=
+
+    # @!method region
+    #   @return [Region]
     storage_forward :region, as: "Region"
+
+    # @!method region=
+    #   @see #region
```

Comment:
NP: to be honest I do not like much this see stuff. Do you consider document it as attribute? http://www.rubydoc.info/gems/yard/file/docs/Tags.md#attribute